.github/workflows/build.yml

```diff
@@ -49,12 +50,12 @@ jobs:
       - name: Install dependencies
         run: npm ci
       - name: Lint check
-        run: npm run lint-check
+        run: npm run lint-check -- -f json -o build/eslint-reporter.json
```

Comment:
:question: why is this needed? if the eslint check fails, the build fails so no other steps are trigger including sonar, so this seems useless

Reply:
ℹ️  This generates the Eslint report, and we need it for Sonar.

Reply:
I understand what you want to say.
If there is a error we don't analyze with Sonar, and if there is no error, sonar display anything. so it's useless.

Maybe, we should failed the job if the lint failed, only at the Sonar step end.